SeaPublicWebsite/Views/EnergyEfficiency/AccessibleLoftSpace.cshtml

```diff
@@ -23,49 +23,49 @@
         
         <form action="@Url.Action("AccessibleLoftSpace_Post", "EnergyEfficiency", new {reference = Model.Reference})" method="post" novalidate>
             @(Html.HiddenFor(m => m.EntryPoint))
-            
             @Html.AntiForgeryToken()
+            
+            @{
+                Func<object, object> details = 
+                    @<text>
+                        <p class="govuk-body">
+                            A loft area is an unheated space without walls and ceilings, immediately beneath the pitched roof. You will usually be able to see the roof timbers.
+                        </p>
+                        <p class="govuk-body">
+                            It is accessible if an installer could easily get into it to do some work – for example, if there is an existing loft hatch and some of the space is big enough for someone to stand up in it.
+                        </p>
+                        <p class="govuk-body">
+                            If all the space under your pitched roof, or nearly all the space, is used as living space with finished walls and ceilings then you don’t have an accessible loft.
+                        </p>
+                     </text>;
+            }
+            
             @(await Html.GovUkRadiosFor(m => m.AccessibleLoftSpace,
                 fieldsetOptions: new FieldsetViewModel
                 {
                     Legend = new LegendViewModel
                     {
-                        Html = @<text>
-                                   <h1 class="govuk-fieldset__heading">
-                                       Do you have an accessible loft area that hasn’t been converted into a room? 
-                                   </h1>
-                                </text>,
-                        Classes = "govuk-fieldset__legend--l"
+                        Text = "Do you have an accessible loft area that hasn’t been converted into a room?",
+                        Classes = "govuk-fieldset__legend--l",
+                        IsPageHeading = true
```

Comment:
As per https://design-system.service.gov.uk/patterns/question-pages/#start-by-asking-one-question-per-page
Single question pages use the legend as heading

Reply:
Nice